src/cli/index.js

```diff
@@ -158,7 +162,7 @@ if (commander.json) {
   Reporter = JSONReporter;
 }
 const reporter = new Reporter({
-  emoji: process.stdout.isTTY && process.platform === 'darwin',
+  emoji: commander.emoji && process.stdout.isTTY && process.platform === 'darwin',
```

Comment:
This is unintuitive. Why `commander.emoji === true`  will _disable_ emojis?
You should'be named it `noEmoji` instead.